DependencyInjection/Configuration.php

```diff
@@ -477,6 +478,7 @@ public function getConfigTreeBuilder()
                             ->scalarNode('title')->defaultNull()->end() // pushover
                             ->scalarNode('host')->defaultNull()->end() // syslogudp & hipchat
                             ->scalarNode('port')->defaultValue(514)->end() // syslogudp
+                            ->scalarNode('rfc')->defaultNull()->end() // syslogudp
```

Comment:
should be an integerNode as SyslogUdpHandler constructor requires an int

Reply:
also please add description and potential values like \Monolog\Handler\SyslogUdpHandler::RFC3164